scrypt.1

```diff
@@ -71,6 +72,8 @@ is given, then
 does not print any prompts, and reads a passphrase from stdin.
 .Sh OPTIONS
 .Bl -tag -width "-m maxmemfrac"
+.It Fl f
+Force the decryption to use any required memory or CPU time.
```

Comment:
I kind-of like the "-f\nForce" bit, though.  What about "Force the decryption to proceed even if it is anticipated to require an excessive amount of memory or CPU time." ?